### DISCUSSION TITLE
Steinberg UR44C

### DISCUSSION THREAD

**Discussion by Ted1957** [jamulussoftware/jamulus#1882] [Technical Support & Help]
Hi all, I bough the Steinberg UR44C Interface to use it for Jamulus. I can route it to virtual driver (Blackhole) and it works. Others told me NOT to use Blackhole, because the interface has a so called Loopback-Function. But when I use this, the others in the room complain about Echoing and the audio of others are going to my channel in Jamulus too. I am using this with Cubase. Anyone the same experiences with this? I guess the "Loopback-Mode" means something different and cannot use with Jamulus ....Thanks for your answers!

**Comment by jujudusud** [comment]
Hi @Ted1957 , could you please tell us which OS and which version you are using ? macOS ? Windows ? Linux ? Which one ?

**Reply by Ted1957** [reply]
Thanks jujudu- I use a Mac Pro with High Sierra....

**Comment by npostavs** [comment]
I have a Motu M2 which also has this loopback feature (I'll assume it works the same as with your interface).

Loopback mode means you can record the sound being played into your interface (as if you held a microphone up to your headphones). So if you're running Jamulus normally, then you could select loopback as input in Cubase, and then you would get the Jamulus session (as your hear it) recorded into Cubase. 

It's not useful for bringing sound back into Jamulus, because Jamulus already plays sound from its inputs to the outputs; you'll end up with echoing as you described (potentially, you could work around it by having Jamulus play into one ear and sounds from Cubase being played into the other, but you might not like the resulting sound).

**Reply by Ted1957** [reply]
Thanks for your answer. So I guess I do have to use Blackhole further...
